modin/pandas/base.py

```diff
@@ -3165,17 +3197,21 @@ def __get_groups(
     @property
     def groups(self):
         return self._query_compiler.default_to_pandas(
-            lambda df: pandas.DataFrame.resample(df, *self.resample_args).groups
+            lambda df: pandas.DataFrame.resample(
+                df, *list(self.resample_args.values())
+            ).groups
         )
 
     @property
     def indices(self):
         return self._query_compiler.default_to_pandas(
-            lambda df: pandas.DataFrame.resample(df, *self.resample_args).indices
+            lambda df: pandas.DataFrame.resample(
+                df, *list(self.resample_args.values())
+            ).indices
         )
 
     def get_group(self, name, obj=None):
-        if self.resample_args[1] == 0:
+        if list(self.resample_args.values())[1] == 0:
```

Comment:
```suggestion
        if self.resample_args.values()[1] == 0:
```

Reply:
@prutskov This not correct, because self.resample_args.values() have type is 'dict_values'

Reply:
@prutskov This is not correct because self.resample_args.values() have type is 'dict_values'